src/git-extract

```diff
@@ -54,17 +54,18 @@ function preconditions {
 }
 
 
-function steps {
-  echo_if_true "stash_open_changes" "$INITIAL_OPEN_CHANGES"
+function should_stash_open_changes {
```

Comment:
Just to think the variable idea through, we could set `STASH_OPEN_CHANGES` to false in `src/helpers/helpers.sh`. Then, if a script wants to have its changes stashed, it could say

```bash
STASH_OPEN_CHANGES=true
```

Git-ship would do:

```bash
if [ "$target_branch_name" != "$INITIAL_BRANCH_NAME" ]; then
  STASH_OPEN_CHANGES=true
fi
```

This would still be shorter and I think more readable.